homeassistant/components/konnected.py

```diff
@@ -107,10 +116,8 @@
             CONF_API_HOST: cfg.get(CONF_API_HOST)
         }
 
-    def device_discovered(service, info):
-        """Call when a Konnected device has been discovered."""
-        host = info.get(CONF_HOST)
-        port = info.get(CONF_PORT)
+    def setup_device(host, port):
+        """Set up a Konnected device at `host` listening on `port`."""
         discovered = DiscoveredDevice(hass, host, port)
         if discovered.is_configured:
             discovered.setup()
```

Comment:
Check for home assistant async api use in this method. I found at least one call to `async_dispatcher_send`, which should be swapped for the sync version.